src/pages/SingleVideo/singleVideo.css

```diff
@@ -0,0 +1,70 @@
+.single-video-page {
+  display: flex;
+  justify-content: space-around;
+  margin-left: 16rem;
+  gap: 1rem;
+  padding: 1rem;
+}
+.video-player {
+  width: 70%;
+}
+.video-player iframe {
+  height: calc(80vh - 72px);
+}
+.suggest-videos {
+  width: 30%;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+.video-info {
+  display: flex;
+  flex-direction: column;
+  gap: 0.75rem;
+}
+.video-title {
+  color: var(--white-color);
+  margin-bottom: 0.25rem;
+}
+.video-creator {
+  color: var(--white-color);
+}
+.user-btns {
+  display: flex;
+  gap: 2rem;
+}
+.user-btn {
+  cursor: pointer;
+  font-size: 1rem;
+  color: var(--white-color);
+  padding: 0.5rem 1rem;
+  background-color: var(--active-color);
+  border-radius: 0.5rem;
+}
+.user-btn:hover {
+  background-color: #1d1d27;
+}
+.user-btn span {
+  margin-left: 8px;
+}
+.video-description p {
+  color: var(--white-color);
+  font-size: 1.2rem;
+  margin: 1rem 0;
```

Comment:
Since you have linked your component library, you can use your predefined CSS variables for spacing here as well.